### PR TITLE
English dialog improvements

### PR DESCRIPTION
**Pull request by craigmaloney**
Added a few English dialog improvements.

### CODE REVIEW

File: en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg [hunk 1]
```diff
@@ -147,12 +147,12 @@
          style="font-style:italic;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:35.77793884px;line-height:1.25;font-family:Lavi;-inkscape-font-specification:'Lavi Italic'"
          x="1856.9244"
          y="283.55594"
-         id="tspan470">public holidays. Offer is</tspan><tspan
+         id="tspan470">public holidays. Offer may</tspan><tspan
          sodipodi:role="line"
          style="font-style:italic;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:35.77793884px;line-height:1.25;font-family:Lavi;-inkscape-font-specification:'Lavi Italic'"
          x="1856.9244"
          y="328.27838"
-         id="tspan472">not cumulative. Limited to</tspan><tspan
```
Comment (agryson) [inline en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg:155]: I opted for cumulative here to make it sound more 'legal', though both can be found in legal blurb - @Deevad you can take your pick

**Review by agryson — APPROVED** [review]
While the cumulative / combined thing may come down to a matter of taste, almost all of these are clear improvements

**Review by agryson — APPROVED** [review]

**Review by agryson — APPROVED** [review]
Original French was capitalized, I assume as a means of emphasis. If we change that we should maybe italicize or emphasize the text in another way to compensate - but not blocking I think.

**Review by agryson — APPROVED** [review]

**Review by agryson — APPROVED** [review]

File: en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg (hunk 1 above)
Comment (ghost) [reply en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg:155]: Nº of results in a search engine, as an indication of usage:

* `offer "not cumulative"`: 371 000
* `offer "may not be combined"`: 46 000 000

**Comment by calimeroteknik** [comment]
A few comments:

Glory should not really be capitalized in French either, but it was, and I let it that way. It's not strictly standard typography but I've seen some RPG and comics do that sort of thing.

I'm not sure it made it in time; the minor adjustment I suggested in French just after the freezing (eep) would reflect as:
```
…and the doorway to Glory will open before you.
```
Or something like that… it might need a fix to something more literary.

**Comment by calimeroteknik** [comment]
About the change that made "idiotic" into "desperate", I think the best match for the original French "idiot" is something like "foolish".

**Comment by ghost** [comment]
@Deevad Do you have a preference?

This might have worked on us before, but...
* we're **no longer idiotic** enough... [original by agryson]
* we're **not desperate** enough... [modified by craigmaloney]
* we're **no longer foolish** enough... [calimeroteknik's applied to agryson's]
* we're **not foolish** enough... [calimeroteknik's applied to craigmaloney's]

to fall for this kind of con game!

**Comment by agryson** [comment]
I prefer "no longer foolish".
It's not as harsh as my original "idiotic" and the "no longer" suggests
that they have grown and changed.

**Comment by craigmaloney** [comment]
I like "no longer foolish". It conveys the original intent, and isn't as harsh as "idiotic".

File: en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg (hunk 1 above)
Comment (craigmaloney) [reply en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg:155]: "may not be combined" is what every USA folk sees on their coupons (paper discounts offered by establishments and marketers in order to entice folks to purchase their wares). So "may not be combined with any other offer" will ring closer to what someone in the USA would see and associate with legal "blah blah blah" along with "offer void where prohibited".

**Comment by Deevad** [comment]
Thank you @agryson , @calimeroteknik , @craigmaloney and @M1dgard for the detailed and constructive feedback. I merged and I'll re-render on XYZ to re-read.

**Comment by ghost** [comment]
Caution @Deevad: the 'no longer foolish' consensus was not yet committed. (So don't do string freeze just yet.)

**Comment by ghost** [comment]
Haha, you beat me to it

**Comment by Deevad** [comment]
@M1dgard : I just commited it in 3830f6c :)  Thank you for the recommendation  :+1:

**Comment by calimeroteknik** [comment]
I think this was lost in the flow:

The minor adjustment https://github.com/Deevad/peppercarrot_ep23_translation/commit/c791ebde957daf17a2e85ce9a3a10a135fc8c7e0 I suggested in French just after the freezing (eep) would reflect as changing:
```
…and the doorway to glory will be open to you!
```
to
```
…and the doorway to glory will open before you!
```
And the capital G was preferred by Deevad I think?

@agryson since I suggested that change to Deevad after the freezing of French I think you translated the previous version. Thoughts on this?

**Comment by Deevad** [comment]
Thanks @calimeroteknik , Yes, you can backport the "Glory" to "glory" on [fr], it's better.

**Comment by Deevad** [comment]
I mean, no need capitalization to Glory/Gloire on both.

**Comment by calimeroteknik** [comment]
I did make the wording more bombastic with the secret intention to be able to remove the capital letter.
I'll wait for Alex's confirmation for the change in wording for the English version.

**Pull request merged**